src/test/java/practice/pageobject/CookieSettingsObjects.java

```diff
@@ -1,36 +1,58 @@
 package practice.pageobject;
 
-import org.openqa.selenium.WebDriver;
 import org.openqa.selenium.WebElement;
 import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import practice.hooks.Hooks;
 
 
 public class CookieSettingsObjects extends BasePageObjects {
 
-    public CookieSettingsObjects(WebDriver driver) {
-        super(driver);
+    public CookieSettingsObjects() {
+        super();
     }
 
     @FindBy(className = "ot-sdk-container")
-    public WebElement cookiePopup;
+    private WebElement cookiePopup;
     @FindBy(className = "cookie-setting-link")
     public WebElement cookiePopupSettings;
```

Comment:
General comment: variables that are only used in the containing class should be private